### ISSUE TITLE
Navigate to website by clicking on website name in the card detail

### ISSUE THREAD

**Issue by The-Phoenics** [abhijeetnishal/Password-Shield#169]
**Is your feature request related to a problem? Please describe.**
It would a better user experience if the user could navigate to the website by simply clicking on the website name in the website-password details card.

**Describe the solution you'd like**
We can make the element (that shows website name in the card) a anchor tag and redirect the user to that website when the user clicks that element.

![anchor-tag](https://github.com/user-attachments/assets/59b2af5e-ce28-41ca-99b8-c194dbc756f7)

**Comment by The-Phoenics** [comment]
Hey @abhijeetnishal 
Can you please take a look at this issue? I would like to work on this under GSSoC'24.

**Comment by abhijeetnishal** [comment]
Hey @The-Phoenics, we need to verify the website also right? 
What if someone add incorrect url and if they redirect it will show error?
Will you verify the added url?

**Comment by The-Phoenics** [comment]
Hey @abhijeetnishal 
Yes we can do a check using `fetch` and resulting status codes. But I think it should be totally up to the user what they want their passwords to map to.

**Comment by abhijeetnishal** [comment]
Yes, it depends on users also. But still add the function to check. 
Assigning this issue to you, good luck.

**Comment by The-Phoenics** [comment]
Hey @abhijeetnishal 
I need your help in setting up the project locally.

This is my server's .env
![2024-07-28_15-36](https://github.com/user-attachments/assets/40993e47-afff-4ea1-93e9-22bbd901b92a)

console:
![problem](https://github.com/user-attachments/assets/07688d74-c143-4248-8dfb-b6ceac596d91)


I could not find the `/client/.env.sample` which should pe pasted and configured in `client/env,sample` as mentioned in Application Setup section of readme

**Comment by abhijeetnishal** [comment]
Hey @The-Phoenics, the env for client is not correct I guess that's why the endpoint is incorrect. The filename for client is local.env and it contains NEXT_PUBLIC_BASE_URL=http://localhost:8080

**Comment by The-Phoenics** [comment]
Hey @abhijeetnishal 
I've added `local.env` in the client directory, still unable to register
![error-env](https://github.com/user-attachments/assets/209dc6ae-2567-4366-ac04-104a49a00698)

**Comment by abhijeetnishal** [comment]
Change the file name to .env.local from local.env and restart the server.
